.eslintignore

```diff
@@ -0,0 +1,2 @@
+prettierrc
+dist
```

Comment:
We should ignore the node_modules folder too

Reply:
done!